### ISSUE TITLE
Implement image compressio

### ISSUE THREAD

**Issue by robustTechie** [robustTechie/ImageX#1]
Add a feature to perform image compression.

**Comment by Gaurav7888** [comment]
Hi I want to work upon this issue.
@robustTechie

**Comment by robustTechie** [comment]
Yes you can work but can you discuss in the slack image compression channel like what did you planned to do

**Comment by Gaurav7888** [comment]
I will use opencv compression  method  and change parameters  which will give best results.

**Comment by Gaurav7888** [comment]
I did some research  because i faced this problem too 
So i would use different  interpolation  methods because it works without introducing  distortions 
Some of the methods are 
Nearest which is give fast and rough results 
Linear, area , cubic , lanczos they are slower but gives better results
Some of them works good for upscale  and some for downscale .

**Comment by robustTechie** [comment]
@Gaurav7888 go ahead :)
